tests/regression/lp-1898038/test-multipass-support-app/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-multipass-support-app
```

Comment:
Same remark about `test-snapd-...` naming scheme.

Reply:
Thanks, fixed in 1deaf49cd7